### PR TITLE
Custom dot cursor added

### PR DESCRIPTION
**Pull request by Amulya-coder**
* Fixes  #872 

  **Work Done:**
* Added cursor.css, cursor.js and update index.html

### CODE REVIEW

File: assets/css/cursor.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
+@import url(https://fonts.googleapis.com/css?family=Open+Sans:400|Raleway:300);
```
Comment (smaranjitghose) [inline assets/css/cursor.css:1]: Why do we need a font for a stylesheet regarding a cursor?

**Comment by smaranjitghose** [comment]
Nice work implementing the cursor so far @Amulya-coder!

## Feedback:

- Reduce the radius of the outer circle to 25% of its current value
- The cursor should be used on all pages except the home page due to the presence of a particle background
- The cursor should not be behind the other elements(Example given below:)

cursro

File: assets/css/cursor.css (hunk 1 above)
Comment (Amulya-coder) [reply assets/css/cursor.css:1]: Thanks for pointing it out! , I will remove it.

**Review by anushbhatia — CHANGES_REQUESTED** [review]
The cursor goes behind some illustration do check that once if it's a bug or a property.

**Comment by Amulya-coder** [comment]
@smaranjitghose I have made the suggested changes, kindly review them.

**Review by smaranjitghose — APPROVED** [review]

**Comment by smaranjitghose** [comment]
Nice work @Amulya-coder 



Congratulations on getting your PR merged 🎉🎉🎉

**Pull request merged**